### PR TITLE
Fill out IANA section

### PR DESCRIPTION
**Pull request by richsalz**
I filled in the text for the IANA Considerations section.

I am not sure if "standards action" is appropriate, but can live with it if you agree.  Expert review seems better, and similar to what other JSON registries do.  Thoughts?

I deleted the comment about I18N because I'm not sure it matters, since it's just JSON. Was that wrong to do?

### CODE REVIEW

File: draft-ietf-tls-wkech.xml [hunk 1]
```diff
@@ -546,18 +546,51 @@ Client <-->|                    | |                    | facing
...
+	    <t>Date Modified:</t>
+	  </list>
+	  Items in curly braces should be replaced with their actual values.
+	</t>
+      </section>
+
+      <section title="JSON HTTP Origin Information">
 
         <t>If approved, this specification requests the creation of an IANA
             registry named "JSON HTTP Origin Info" with a Standards Action
```
[the remark this answers is not in the document]
Comment (richsalz) [reply draft-ietf-tls-wkech.xml]: fixup commit pushed that does the three renamings in the doc.

[the remark this answers is not in the document]
Comment (richsalz) [reply draft-ietf-tls-wkech.xml]: I dunno, I would think #14 might add things?

[the remark this answers is not in the document]
Comment (bemasc) [reply draft-ietf-tls-wkech.xml]: I think that's just another SvcParamKey.

[the remark this answers is not in the document]
Comment (richsalz) [reply draft-ietf-tls-wkech.xml]: Well, we might move regeninterval to the top-level :)

**Comment by richsalz** [comment]
And I added the registry entry for the top-level `regeninterval` field, as discussed in #28

**Review by bemasc — APPROVED** [review]

**Pull request merged**